src/components/ButtonGroup/ButtonGroup.js

```diff
@@ -56,8 +56,8 @@ const ButtonGroupList = styled('ul')(baseStyles, alignmentStyles);
 /**
  * Groups its Button children into a list and adds margins between.
  */
-const ButtonGroup = ({ children, align, noMargin }) => (
-  <ButtonGroupList align={align} noMargin={noMargin}>
+const ButtonGroup = ({ children, align, noMargin, className }) => (
+  <ButtonGroupList className={className} align={align} noMargin={noMargin}>
```

Comment:
Shouldn't we just forward otherProps (like `...rest`)? what if we want a data-attr for instance